docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -0,0 +1,175 @@
+# Autofill Manager
+
+The Tizen autofill framework allows more than one or more autofill services to be installed on a Tizen device. Autofill Manager is a module for managing the installed autofill services. Application developers can use Autofill manager to get the current autofill service or change the autofill service.
```

Comment:
@imfine98 
Global question
In this PR, "autofill" and "Autofill" are mixed, as like "Autofill manager", "autofill manager", "autofill service" or "Autofill service".
I checked the tmp API file for 5.5 but it also shows mixed upper & small letters.
Is there no upper & small letter rule to use the "Autofill"?

Reply:
@mijin85cho, @sabeena21,
I followed the Sabeena's comment. Is there any guideline for this case?
Which one is the appropriate?
1. Autofill Manager
2. Autofill manager
3. autofill manager

I am also confused. I'll follow your guide if you'll suggest.

Reply:
@imfine98,
Depends on the scenarios:
Scenario 1: If it is a title then"Autofill Manager is written as it is. In title case.
Scenario 2: If we are referring to something that appears in the UI, then as it appears  as it in the UI.
Scenario 3: If it is just in a sentence then we can use autofill manager. 
Scenario 4: If "Autofill Manager" is been defined as Autofill Manager. Then we need to exactly write it in the same manner.
For example, JDK is defined as JDK and it cannot be changed.
If Autofill Manager is represented as "Autofill Manager" then we will write exactly as it is.

Reply:
Fixed. Please check again. Please let me know if you will find the wrong usage.